frontside/android/build.gradle

```diff
@@ -2,7 +2,7 @@ buildscript {
     ext.kotlin_version = '1.3.50'
     repositories {
         google()
-        jcenter()
+        mavenCentral()
```

Comment:
Why? I don't have any experience with maven repos but according to this:
<img width="730" alt="image" src="https://user-images.githubusercontent.com/85256521/140644281-292c1adb-a56d-454e-8480-9509ad02d663.png">

I would choose `jcenter()`.

source: https://stackoverflow.com/a/50726436/10459242

Reply:
change to `jcenter()`